ofproto/ofproto-dpif.c

```diff
@@ -2220,6 +2227,14 @@ port_destruct(struct ofport *port_, bool del)
     }
 
     tnl_port_del(port, port->odp_port);
+#if defined(P4OVS)
+    if (port->is_tunnel) {
+        struct tunnel_info tnl_info;
+        memset(&tnl_info, 0, sizeof(tnl_info));
+        get_tunnel_data(port->up.netdev, &tnl_info);
```

Comment:
Should this function continue if `get_tunnel_data` returns an error?

Reply:
Added a check